google/import_compute_shared_vpc_test.go

```diff
@@ -0,0 +1,41 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccComputeSharedVpc_importBasic(t *testing.T) {
```

Comment:
we're actually trying to get away from having separate files for import tests, because it hides the fact that we're actually duplicating a lot of our tests. Instead, can you add the TestSteps that do the import to the regular tests?

Reply:
Will do.